scripts/world_bank/health_nutrition_population/HealthcareExpenditure_README.md

```diff
@@ -0,0 +1,83 @@
+Importing World Bank Healthcare Expenditure Data into Data Commons
+Author: aarushitiwari@
+
+About the Dataset
+The world bank HNP dataset contains information about population, nutrition and health statistics of all countries, updated annually from all countries and marked special regions 
+The dataset from 1960 onwards is available online. For this import, the series selected are all population statistics – male and female, age 00 to 25.
+
+
+Size
+?
+
+Format: Json (access is directly from browser)
+
+Time period: Annual, from 1960
+
+Granularity: Country level
+Variables
+
+
+The data set can be downloaded from  with the following API:
+An example of series would be “'SH.SGR.CRSK.ZS', containing information about the % of people at risk of catastrophic expenditure on surgery.
+
+
+
+https://api.worldbank.org/v2/country/all/indicator/{series}?format=JSON&per_page=17000
+
```

Comment:
is there a script you used to download these files?

please also add instructions here for how to run these scripts to generate the final outputs.